astropy/io/misc/asdf/tags/transform/tabular.py

```diff
@@ -60,12 +61,24 @@ def assert_equal(cls, a, b):
         if isinstance(a.lookup_table, u.Quantity):
             assert u.allclose(a.lookup_table, b.lookup_table)
             assert u.allclose(a.points, b.points)
-            for i in range(len(a.bounding_box)):
-                assert u.allclose(a.bounding_box[i], b.bounding_box[i])
+            a_box = a.bounding_box
```

Comment:
Hmm... Isn't @eslavich taking out the `asdf` stuff soon? Is this also reflected in the new package that he is working on (or something)?

Reply:
Yes he is, I have been working with him on this.